lib/puppet/util/windows/security.rb

```diff
@@ -247,11 +246,9 @@ def get_mode(path)
       # if owner and group the same, then user and group modes are the OR of both
       if sd.owner == sd.group
         mode |= ((mode & S_IRWXG) << 3) | ((mode & S_IRWXU) >> 3)
-        #puts "owner: #{sd.group}, 0x#{ace.mask.to_s(16)}, #{mode.to_s(8)}"
```

Comment:
These were actually in there for devs to uncomment while debugging... I think @joshcooper originally added them, but I've used them from time to time.